miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -19,12 +19,13 @@ abstract class MiniApp internal constructor() {
     abstract suspend fun listMiniApp(): List<MiniAppInfo>
 
     /**
-     * Creates a mini app for the given [appId]. The mini app is downloaded and saved.
-     * Provides [MiniAppView] when successful and an [error] when there is some issue
-     * during fetching, downloading or creating a view.
+     * Creates a mini app from the metadata [MiniAppInfo] object.
+     * The mini app is downloaded, saved and provides a [MiniAppView] when successful
+     * @throws MiniAppSdkException when there is some issue during fetching,
+     * downloading or creating the view.
      */
     abstract suspend fun create(
-        appId: String,
+        miniAppInfo: MiniAppInfo,
```

Comment:
I see it as something which is temporary because of the changing specs at the BE.
The current implementation is just because of the access situation with versionId.
This is definitely going to change going forward.

Reply:
I don't think any API changes are going to affect this at this point. The `ListingApi` is the only way that we will have to retrieve the latest version ID.

But I think for now, we don't necessarily need the ability to create a Mini App when provided only the App ID so probably this is okay to leave it as it is (although this will be a requirement in the future).